modules/mlflow/mlflow-fargate/tests/test_stack.py

```diff
@@ -71,6 +72,20 @@ def test_synthesize_stack(stack_defaults, use_rds) -> None:
         ),
     )
 
+
+@pytest.mark.parametrize("use_rds", [False, True])
+def test_synthesize_stack(stack: cdk.Stack, use_rds: bool) -> None:
     template = Template.from_stack(stack)
     template.resource_count_is("AWS::ECS::Cluster", 1)
     template.resource_count_is("AWS::ECS::TaskDefinition", 1)
+
+
+@pytest.mark.parametrize("use_rds", [False, True])
+def test_no_cdk_nag_errors(stack: cdk.Stack, use_rds: bool) -> None:
```

Comment:
Thanks, this is great. CDK nag runs at synth, but we didn't have a way to capture those errors when PRs are raised unless we deploy the modules manually.